src/V3Premit.cpp

```diff
@@ -173,16 +194,50 @@ class PremitVisitor final : public AstNVisitor {
         AstNRelinker linker;
         nodep->unlinkFrBack(&linker);
 
-        AstVar* varp = getBlockTemp(nodep);
+        AstVar* varp;
+
```

Comment:
```suggestion
        AstVar* varp = nullptr;

```

I realize right now it's always assigned, but perhaps we make a change someday.